common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -455,3 +459,47 @@ def test_teams_membership(self):
             self.browse_teams_page.team_cards[0].find_element_by_css_selector('.member-count').text,
             '1 / 10 Members'
         )
+
+
+@attr('shard_5')
+class TeamPageTest(TeamsTabBase):
+    """Tests for viewing a specific team"""
+    def setUp(self):
+        super(TeamPageTest, self).setUp()
+        self.topic = {u"name": u"Example Topic", u"id": "example_topic", u"description": "Description"}
+        self.set_team_configuration({'course_id': self.course_id, 'max_team_size': 10, 'topics': [self.topic]})
+        self.team = self.create_teams(self.topic, 1)[0]
+        self.create_membership(self.user_info['username'], self.team['id'])
+        self.team_page = TeamPage(self.browser, self.course_id, self.team)
+
+    def setup_thread(self):
+        """
+        Set up multiple threads on the team page by passing 'thread_count'.
+        """
+        thread = Thread(
+            id="test_thread_{}".format(uuid4().hex),
+            commentable_id=self.team['discussion_topic_id'],
+            body="Dummy text body."
+        )
+        thread_fixture = MultipleThreadFixture([thread])
+        thread_fixture.push()
+        return thread
+
+    def test_discussion_on_team_page(self):
+        """
+        Scenario: Team Page renders a team discussion.
+        Given I am enrolled in a course with a team configuration, a topic,
+            and a team belonging to that topic
+        When I post a thread in the team's discussion
```

Comment:
Nit: this test doesn't actually post a thread as any particular user. I would say

When a thread exists in the team's discussion 
...
And I should see the existing thread